ote/src/cljs/ote/views/parking.cljs

```diff
@@ -113,31 +113,26 @@
                    {:name  ::t-service/all-day
                     :width "10%"
                     :type  :checkbox
-                    :write (fn [data all-day?]
+                    :write (do
+                             #(assoc-in %1 [::t-service/service-hours ::t-service/all-day] %2)(fn [data all-day?]
```

Comment:
Why is assoc-in here?

Reply:
yes. it didn't affect I thought that if :write parameter is given it will override default functionality. Thanx.
Removed now